### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/778

### CODE REVIEW

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ contract MyToken is ERC20, Ownable {
 
 至此，我们已经使用OpenZeppelin合约和库完成了ERC-20代币合约的部署。接下来，我们就可以通过Remix与代币合约进行交互，或者将合约添加到MetaMask。
 
-### 部署ERC-721代币 {: #deploying-an-erc721-token } 
+### 部署ERC-721代币 {: #deploying-an-erc721-token }
 
 在本示例中，我们将向Moonbase Alpha部署一枚ERC-721代币。最终我们所使用的代码结合了从OpenZeppelin中抽取的不同合约：
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:117]: Line 125 below this, ERC-20 link is broken and should be updated to: /builders/build/eth-api/dev-env/openzeppelin/contracts/#deploying-an-erc20-token

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md [hunk 2]
```diff
@@ -177,7 +177,7 @@ contract MyToken is ERC721, ERC721Enumerable, ERC721Burnable, Ownable {
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:177]: Line 166 Contract Wizard link should be updated to builders/build/eth-api/dev-env/openzeppelin/contracts/#openzeppelin-contract-wizard

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md [hunk 3]
```diff
@@ -5,57 +5,57 @@ description: 了解如何使用 OpenZeppelin Contracts Wizard创建常见智能
...
-## OpenZeppelin Contract Wizard {: #openzeppelin-contract-wizard } 
+## OpenZeppelin Contract Wizard {: #openzeppelin-contract-wizard }
 
-OpenZeppelin开发了一种基于网络的线上智能合约交互式工具，它可能是使用OpenZeppelin代码编写智能合约最简单快捷的方式。这一工具称为[Contracts Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard){target=_blank}。
+OpenZeppelin开发了一种基于网络的线上智能合约交互式工具，它可能是使用OpenZeppelin代码编写智能合约最简单快捷的方式。这一工具称为[Contracts Wizard](https://docs.openzeppelin.com/contracts/5.x/wizard){target=_blank}。
 
 目前Contracts Wizard支持以下ERC标准：
 
  - [**ERC-20**](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/){target=_blank} —— 遵守[EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}的同质化代币标准。同质化指的是所有代币均相同，并可互换，也就是说它们具有相同的价值。同质化代币的一个典型例子就是法币，面值相等的纸币其价值相等
  - [**ERC-721**](https://ethereum.org/en/developers/docs/standards/tokens/erc-721/){target=_blank} —— 遵守[EIP-721](https://eips.ethereum.org/EIPS/eip-721){target=_blank}的非同质化代币合约。非同质化指的是每一枚代币都不一样，因此独一无二。ERC-721代币可以用于代表某一特定物品的所有权，例如游戏、房地产中的收藏品等
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:23]:
```suggestion
 - [**ERC-721**](https://ethereum.org/zh/developers/docs/standards/tokens/erc-721/){target=_blank} —— 遵守[EIP-721](https://eips.ethereum.org/EIPS/eip-721){target=_blank}的非同质化代币合约。非同质化指的是每一枚代币都不一样，因此独一无二。ERC-721代币可以用于代表某一特定物品的所有权，例如游戏、房地产中的收藏品等
```

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md [hunk 4]
```diff
@@ -5,57 +5,57 @@ description: 了解如何使用 OpenZeppelin Contracts Wizard创建常见智能
...
 
-## OpenZeppelin Contract Wizard {: #openzeppelin-contract-wizard } 
+## OpenZeppelin Contract Wizard {: #openzeppelin-contract-wizard }
 
-OpenZeppelin开发了一种基于网络的线上智能合约交互式工具，它可能是使用OpenZeppelin代码编写智能合约最简单快捷的方式。这一工具称为[Contracts Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard){target=_blank}。
+OpenZeppelin开发了一种基于网络的线上智能合约交互式工具，它可能是使用OpenZeppelin代码编写智能合约最简单快捷的方式。这一工具称为[Contracts Wizard](https://docs.openzeppelin.com/contracts/5.x/wizard){target=_blank}。
 
 目前Contracts Wizard支持以下ERC标准：
 
  - [**ERC-20**](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/){target=_blank} —— 遵守[EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}的同质化代币标准。同质化指的是所有代币均相同，并可互换，也就是说它们具有相同的价值。同质化代币的一个典型例子就是法币，面值相等的纸币其价值相等
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:22]:
```suggestion
 - [**ERC-20**](https://ethereum.org/zh/developers/docs/standards/tokens/erc-20/){target=_blank} —— 遵守[EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}的同质化代币标准。同质化指的是所有代币均相同，并可互换，也就是说它们具有相同的价值。同质化代币的一个典型例子就是法币，面值相等的纸币其价值相等
```

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md [hunk 5]
```diff
@@ -177,7 +177,7 @@ contract MyToken is ERC721, ERC721Enumerable, ERC721Burnable, Ownable {
 
 至此，我们已经使用OpenZeppelin合约和库完成了ERC-721代币合约的部署。接下来，我们就可以通过Remix与代币合约进行交互，或者将合约添加到MetaMask。
 
-### 部署ERC-1155代币 {: #deploying-an-erc1155-token } 
+### 部署ERC-1155代币 {: #deploying-an-erc1155-token }
 
 在本示例中，我们将向Moonbase Alpha部署一枚ERC-1155代币。最终我们所使用的代码结合了从OpenZeppelin中抽取的不同合约：
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:183]: Line 192 link also needs to updated to /builders/build/eth-api/dev-env/openzeppelin/contracts/#deploying-an-erc20-token

File: builders/build/eth-api/dev-env/openzeppelin/contracts.md
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/contracts.md:1]: Contract Wizard link on Line 229 should be updated to: builders/build/eth-api/dev-env/openzeppelin/contracts/#openzeppelin-contract-wizard

File: builders/build/eth-api/dev-env/openzeppelin/defender.md [hunk 6]
```diff
@@ -5,24 +5,24 @@ description:  通过此教程学习如何使用OpenZeppelin Defender在兼容以
...
 
-## 开始使用Defender {: #getting-started-with-defender } 
+## 开始使用Defender {: #getting-started-with-defender }
 
 本小节将介绍在Moonbase Alpha上开始使用OpenZeppelin的步骤。
 
-### 检查先决条件 {: #checking-prerequisites } 
+### 检查先决条件 {: #checking-prerequisites }
 
 请确保安装[MetaMask](https://metamask.io/){target=_blank}，并已连接至Moonbase Alpha测试网。如果您还没有将MetaMask连接到测试网，请根据我们的[MetaMask整合教程](/integrations/wallets/metamask/){target=_blank}进行操作。
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/defender.md:27]:
```suggestion
请确保安装[MetaMask](https://metamask.io/){target=_blank}，并已连接至Moonbase Alpha测试网。如果您还没有将MetaMask连接到测试网，请根据我们的[MetaMask整合教程](/builders/integrations/wallets/metamask/){target=_blank}进行操作。
```

File: builders/build/eth-api/dev-env/openzeppelin/defender.md [hunk 7]
```diff
@@ -70,19 +70,19 @@ contract PausableBox is Initializable, PausableUpgradeable, OwnableUpgradeable {
 }
 ```
 
-### 连接Defender {: #connecting-defender } 
+### 连接Defender {: #connecting-defender }
 
 注册了OpenZeppelin Defender账户以后，登录[Defender App](https://defender.openzeppelin.com/){target=_blank}。在主屏幕中，您将看到[MetaMask已连接到Moonbase Alpha](/getting-started/moonbase/metamask/){target=_blank}，点击右上角的**Connect wallet**（连接钱包）按钮：
```
Comment (themacexpert) [inline builders/build/eth-api/dev-env/openzeppelin/defender.md:75]:
```suggestion
注册了OpenZeppelin Defender账户以后，登录[Defender App](https://defender.openzeppelin.com/){target=_blank}。在主屏幕中，您将看到[MetaMask已连接到Moonbase Alpha](/builders/get-started/networks/moonbase/#connect-metamask){target=_blank}，点击右上角的**Connect wallet**（连接钱包）按钮：
```

File: builders/interoperability/xcm/xc-registration/xc-integration.md
Comment (themacexpert) [inline builders/interoperability/xcm/xc-registration/xc-integration.md:1]: Line 256 has a typo, should be: https://github.com/Moonsong-Labs/xcm-tools

File: learn/platform/glossary.md [hunk 8]
```diff
@@ -7,54 +7,54 @@ description: 我们整理了与波卡相关的中英对照术语表，助您了
...
 
-### 中继链 （Relay Chain） {: #relay-chain } 
+### 中继链 （Relay Chain） {: #relay-chain }
 
 中继链是波卡（Polkadot）网络的中心链。平行链连接到中继链，并将其用于共享安全性和消息传递。中继链上的验证人有助于保护平行链。
 
-### 智能合约 （Smart Contract） {: #smart-contract } 
+### 智能合约 （Smart Contract） {: #smart-contract }
 
 智能合约是一种计算机程序或交易协议，旨在自动执行合约或协议的条款，控制或记录与法律相关的事件和动作。智能合约的总体目标是满足共同的合同条件（例如付款项、留置权、保密性，甚至强制执行），最大限度地减少异常以及对可信中介的需求。 相关的经济目标包括减少欺诈损失、仲裁、执行成本以及其他交易成本。[点击此处了解更多](https://en.wikipedia.org/wiki/Smart_contract){target=_blank}。
```
Comment (themacexpert) [inline learn/platform/glossary.md:44]:
```suggestion
智能合约是一种计算机程序或交易协议，旨在自动执行合约或协议的条款，控制或记录与法律相关的事件和动作。智能合约的总体目标是满足共同的合同条件（例如付款项、留置权、保密性，甚至强制执行），最大限度地减少异常以及对可信中介的需求。 相关的经济目标包括减少欺诈损失、仲裁、执行成本以及其他交易成本。[点击此处了解更多](https://zh.wikipedia.org/wiki/%E6%99%BA%E8%83%BD%E5%90%88%E7%BA%A6){target=_blank}。
```

**Review by themacexpert — APPROVED** [review]
Noticed a couple broken links and in other places suggested the Chinese translation for certain links to wikipedia, etc.

**Comment by eshaben** [comment]
Good catches! Thanks!

**Pull request merged**